packages/dapp/src/components/listinglist/Listings.tsx

```diff
@@ -55,7 +65,7 @@ class Listings extends React.Component<ListingProps & ListingReduxProps> {
           TabComponent={StyledTabLarge}
           onActiveTabChange={this.onTabChange}
         >
-          <Tab title={"Whitelisted Newsrooms"}>
+          <Tab title={<ApprovedNewsroomsTabText />}>
             <StyledPageContent>
               <StyledListingCopy>
                 All approved Newsrooms should align with the Civil Constitution, and are subject to Civil community
```

Comment:
I guess we should prob put the tab copy (ie: "All approved Newsrooms...") into `textComponents` somewhere